app/jobs/in_person/email_reminder_job.rb

```diff
@@ -1,5 +1,8 @@
 module InPerson
   class EmailReminderJob < ApplicationJob
+    EMAIL_TYPE_EARLY = 'early'.freeze
+    EMAIL_TYPE_LATE = 'late'.freeze
```

Comment:
if we wanted, we could add the magic comment so we don't need to freeze strings individually

```suggestion
# frozen_string_literal: true

module InPerson
  class EmailReminderJob < ApplicationJob
    EMAIL_TYPE_EARLY = 'early'
    EMAIL_TYPE_LATE = 'late'
```

Reply:
done in 06e485d12f6e0904fbf0ff6a85b63efb61268c34